src/ProjectService.Data/TaskPropertyRepository.cs

```diff
@@ -54,7 +54,7 @@ public IEnumerable<DbTaskProperty> Find(FindTaskPropertiesFilter filter, int ski
                 throw new BadRequestException("Skip count can't be less than 0.");
             }
 
-            if (takeCount <= 0)
+            if (takeCount < 1)
             {
                 throw new BadRequestException("Take count can't be equal or less than 0.");
```

Comment:
Need change description

Reply:
this messages mean one thing, but okey, i'll fix it